.github/workflows/inductor-perf-compare.yml

```diff
@@ -13,30 +13,42 @@ concurrency:
 permissions: read-all
 
 jobs:
-  get-label-type:
-    name: get-label-type
+  get-build-label-type:
```

Comment:
nit: Feels kinda weird to call this the build label since this is technically the default, cross-fleet label.  It makes sense in the context of this file, but this isn't a convention that can be copied over to other workflows

Naming things is hard

Reply:
should we just keep the job name as `get-label-type`? 

Feels confusing to me TBH

Reply:
Perhaps `default-label-prefix` could be a new convention?